lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -825,7 +825,7 @@ def test_grades_csv(self):
         headers = rows[0]
 
         # picking first student records
-        data = dict(zip(headers.strip().split(','), rows[1].strip().split(',')))
+        data = dict(zip(headers.strip().split(','), rows[2].strip().split(',')))
```

Comment:
Does that change if you run just this one test? I'm wondering if it's either a test isolation issue (where we're getting side-effects from another test in the suite), or the ordering of the data is undefined because there's no explicit sorting on it...